src/main/scala/mesosphere/marathon/core/storage/store/impl/cache/LazyCachingPersistenceStore.scala

```diff
@@ -101,8 +107,13 @@ class LazyCachingPersistenceStore[K, Category, Serialized](
           Future.successful(v)
         case _ =>
           async { // linter:ignore UnnecessaryElseBranch
-            val value = await(store.get(id))
+            val value: Option[V] = await(store.get(id))
```

Comment:
I don't see anything that adds clarity - the annotation is unnecessary - you can use tooling to provide a visual type annotation in this case.